### ISSUE TITLE
SA1009 is reported for conditional access

### ISSUE THREAD

**Issue by pdelvo** [DotNetAnalyzers/StyleCopAnalyzers#374]
In this case SA1009 is reprted at each ()?., because ) is not followed by a space:
```csharp
token.GetLocation()?.GetMappedLineSpan()
```

**Comment by sharwell** [comment]
:expressionless:
